.kokoro/client-library-check.sh

```diff
@@ -42,13 +88,26 @@ mvn install -DskipTests=true -Dmaven.javadoc.skip=true -Dgcloud.download.skip=tr
 # Read the current version of this BOM in the POM. Example version: '0.116.1-alpha-SNAPSHOT'
 VERSION_POM=java-shared-config/pom.xml
 # Namespace (xmlns) prevents xmllint from specifying tag names in XPath
-VERSION=`sed -e 's/xmlns=".*"//' ${VERSION_POM} | xmllint --xpath '/project/version/text()' -`
+JAVA_SHARED_CONFIG_VERSION=`sed -e 's/xmlns=".*"//' ${VERSION_POM} | xmllint --xpath '/project/version/text()' -`
 
-if [ -z "${VERSION}" ]; then
+if [ -z "${JAVA_SHARED_CONFIG_VERSION}" ]; then
   echo "Version is not found in ${VERSION_POM}"
   exit 1
 fi
-echo "Version: ${VERSION}"
+echo "Version: ${JAVA_SHARED_CONFIG_VERSION}"
+
+# Update java-shared-config in sdk-platform-java-config
+git clone "https://github.com/googleapis/sdk-platform-java.git" --depth=1
+pushd sdk-platform-java
+SDK_PLATFORM_JAVA_CONFIG_VERSION=$(get_current_version_from_versions_txt versions.txt "google-cloud-shared-dependencies")
+RELEASED_SHARED_DEPENDENCIES_VERSION=$(get_released_version_from_versions_txt versions.txt "google-cloud-shared-dependencies")
+pushd sdk-platform-java-config
+
+# Only update java-shared-config but keep java-shared-dependencies at the release version
```

Comment:
Could you help explain why we want there to be two different versions here?

Reply:
That's a good question. The sdk-platform-java-config artifact holds `google-cloud-shared-dependencies` version in a property variable which is then used by downstream repos ([link](https://github.com/googleapis/java-pubsub/blob/9d7ad1a9c29eef48750bb163ad90eed2dbde4380/pom.xml#L64)). However, since we are updating the version of sdk-platform-java-config in the downstream library, we may have a situation where the SNAPSHOT version of the google-cloud-shared-dependencies also gets updated. This could result in a false positive result where the check could fail due to unreleased changes in shared-dependencies even though java-shared-config is fine.

Reply:
Ah got it, thanks for explaining!